src/libs/ReportUtils.ts

```diff
@@ -2195,9 +2196,6 @@ function getReimbursementDeQueuedActionMessage(
     reportOrID: OnyxEntry<Report> | EmptyObject | string,
     isLHNPreview = false,
 ): string {
-    if (!ReportActionsUtils.isReimbursementDeQueuedAction(reportAction)) {
-        return '';
-    }
     const report = typeof reportOrID === 'string' ? allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportOrID}`] : reportOrID;
```

Comment:
I don't think we should add this checks if we have types that will complain when we call these functions with the wrong reportAction